Sources/ObjCSupport/Builders/LOKStackLayoutBuilder.h

```diff
@@ -8,6 +8,23 @@
 
 #import "LOKBaseLayoutBuilder.h"
 
+typedef NS_ENUM(NSInteger, LOKStackLayoutDistribution) {
+    LOKStackLayoutDistributionDefault = 0,
```

Comment:
You don't need to define that it's = 0. This is the default behavior. Also you don't need to define any associated value because the extension just converts it to swift enum anyway right? Same comment for the other NS_ENUMs

Reply:
True. TBH I just copied these out of the LayoutKitObjC-Swift.h auto-generated file, I can clean these up.